purchase_invoicing_no_zero_line/views/account_view.xml

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <data>
         <record id="view_account_journal_form_inh_no_zero" model="ir.ui.view">
```

Comment:
Also nice to have, could you please de-indent the whole `record` so the XML still looks nice again?

Reply:
Done. I was actually convinced pre-commit was in charge of such changes in the xml files...

Reply:
:+1: